ddtrace/contrib/internal/requests/constants.py

```diff
@@ -1 +1,2 @@
 DEFAULT_SERVICE = "requests"
+USER_AGENT_HEADER = "User-Agent"
```

Comment:
nit: since it looks like the constant is declared here too, maybe it's worth consolidating with the one used for grpc?